src/main/java/com/o19s/es/ltr/feature/LtrModel.java

```diff
@@ -12,22 +12,31 @@
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
- *
  */
-package com.o19s.es.ltr.query;
 
-import ciir.umass.edu.learning.Ranker;
-import org.apache.lucene.queries.CustomScoreQuery;
-import org.apache.lucene.search.Query;
+package com.o19s.es.ltr.feature;
+
+import com.o19s.es.ltr.ranker.LtrRanker;
 
 /**
- * Created by doug on 12/23/16.
+ * Represents a self contained LTR model
  */
-public class RankLibQuery extends CustomScoreQuery {
+public interface LtrModel {
+    /**
+     * Name of the model
+     * @return the name
```

Comment:
Sure, in fact I can remove some comment lines, javadoc complained about the empty `@return` line but is happy if I remove it completely.